src/EditorFeatures/CSharpTest/PdbSourceDocument/PdbSourceDocumentTests.cs

```diff
@@ -457,6 +457,50 @@ public class C
     {
     }
 }
+";
+
+            await RunTestAsync(async path =>
```

Comment:
The diff on this is weird. This test is actually old, just with a few bits of code removed. This wasn't testing type forwards at all, but their presence made me think I was correctly testing them.. which I wasn't!